Algorithms/Sorting /Bubble_Sort.cpp

```diff
@@ -0,0 +1,46 @@
+#include <bits/stdc++.h>
+using namespace std;
+#define ll long long //using macro definition (directive) 
```

Comment:
use int/long.
remove macros.

Reply:
indentation issues.
have a space after a for and if example: for (int i = 0; i < n; i ++)